homeassistant/components/esphome/assist_satellite.py

```diff
@@ -79,6 +79,7 @@
 )
 
 _ANNOUNCEMENT_TIMEOUT_SEC = 5 * 60  # 5 minutes
+_CONFIG_TIMEOUT_SEC = 5
```

Comment:
Where is this used?

Reply:
Leftovers from a parallel universe :smile: 
Removed from here (added in a different PR)